stock_picking_return_refund_option/views/stock_picking_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="view_picking_form" model="ir.ui.view">
+    <field name="inherit_id" ref="stock.view_picking_form"/>
+    <field name="model">stock.picking</field>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='owner_id']" position="after">
+            <field name="to_refund_lines"
+                   attrs="{'invisible': [('picking_type_code', '!=', 'incoming')]}"
```

Comment:
Instead of this condition, you should put one that hides it when this is not a return.